bigip/resource_bigip_ltm_monitor.go

```diff
@@ -148,10 +170,12 @@ func resourceBigipLtmMonitorRead(d *schema.ResourceData, meta interface{}) error
 			d.Set("manual_resume", m.ManualResume)
 			d.Set("parent", m.ParentMonitor)
 			d.Set("name", name)
+			d.Set("username", m.Username)
+			d.Set("password", d.Get("password").(string))
```

Comment:
The test passes if I set the state file equal to the input (`d.Set("password", d.Get("password").(string))`). I can't find any public documentation on how the monitor hash is generated - they're prefixed with `$M$` which I can't find documentation on anywhere. I can certainly ask F5 if they'll provide me with whatever function generates the hash?